src/lib/import-export/tests/export/exporters/default-exporter.test.ts

```diff
@@ -31,6 +37,13 @@ jest.mock( 'archiver', () => {
 	return jest.fn( () => createMockArchiver() );
 } );
 
+// Mock SiteServer
+jest.mock( '../../../../../site-server' );
+( SiteServer.get as jest.Mock ).mockReturnValue( {
+	details: { path: '/path/to/site' },
+	executeWpCliCommand: jest.fn().mockReturnValue( { stderr: null } ),
+} );
```

Comment:
We have other implementation mocks defined in the `beforeEach` section (i.e. test setup). This is a common practice in Jest, so it would be great to apply it for this mock too.